pkgs/tools/filesystems/irods/icommands.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv, fetchFromGitHub, bzip2, zlib, avro-cpp, jansson, zeromq, cppzmq, openssl, kerberos, cmake, gnumake, llvmPackages, boost, libarchive, irods }:
+
+stdenv.mkDerivation rec {
+  version = "git-2016-10-24";
+  name = "irods-icommands-${version}";
+ 
+  buildInputs = [ bzip2 zlib cmake gnumake cppzmq libarchive avro-cpp jansson zeromq openssl llvmPackages.libcxx kerberos boost irods ];
+
+  propagateBuildInputs = [ boost ];
+
+  enableParallelBuilding = true;
+
+  src = [ ( fetchFromGitHub {
```

Comment:
Indentation doesn't feel idiomatic here. Also `src` is typically a single value, whereas `srcs` gets treated as a list.